Makefile

```diff
@@ -86,7 +86,10 @@ dependencies:
 
 # Generate code, CRDs and documentation
 ALL_CRDS=config/crds/all-crds.yaml
-generate: generate-crds generate-api-docs generate-notice-file
+generate: tidy generate-crds generate-api-docs generate-notice-file
+
+tidy:
```

Comment:
I'm fine with `tidy` but wonder if `sync-go-deps` would be better to describe what `go mod tidy` does.

Reply:
🤷‍♀️ at least for me i already know `go mod tidy` and would have to think about it if i read sync-go-deps. Maybe let's leave it as is for now and if we add more things like this we will change it? I don't feel strongly either way though